lusee/LBeam_Gauss.py

```diff
@@ -0,0 +1,47 @@
+from math import degrees
+from .LBeam import LBeam
+import numpy as np
+
+def gauss_beam(theta, theta_fwhm):
+    """
+    Creates a map-level gaussian beam centered at zero
+
+    for a gaussian, fwhm = 2.355*sigma
+    """
+
+    return 2.355/(np.sqrt(2*np.pi)*theta_fwhm)*np.exp(-2.355**2 * theta**2 / (2*theta_fwhm**2) )
```

Comment:
Thisl looks wrong. You want to create a gaussian centered around some theta and phi=0 and have sigma extend in both theta and phi directions. Ofc correctly taking into account sin(theta) factor is good. So you need something like (theta-theta_c) at min.